justfile

```diff
@@ -19,11 +19,10 @@ test-unit:
 bootstrap:
     go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.32
     go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@v1.3
-    go install github.com/oapi-codegen/oapi-codegen/v2/cmd/oapi-codegen@v2.3
```

Comment:
We pinned this to a version to avoid unnecessary codegen updates. It's fine to update but maybe a specific version instead of latest.

Reply:
Thanks for the catch, I mean to pin to `v2.3.0` which is the version they swapped to the new `github.com` location.